corehq/apps/api/resources/v0_5.py

```diff
@@ -1390,3 +1391,42 @@ def get_datasource_data(request, config_id, domain):
     query = cursor_based_query_for_datasource(request_params, datasource_adapter)
     data = response_for_cursor_based_pagination(request, query, request_params, datasource_adapter)
     return JsonResponse(data)
+
+
+class CommCareAnalyticsUserResource(CouchResourceMixin, HqBaseResource, DomainSpecificResourceMixin):
+
+    roles = fields.ListField()
+    permissions = fields.DictField()
+
+    class Meta(CustomResourceMeta):
+        resource_name = 'analytics-roles'
+        list_allowed_methods = []
```

Comment:
Does this matter anymore since we used the "prepend_urls" to override the default list method? I am unsure if we are disabling list method by setting this to `[]` and then also overriding it later?

Reply:
Good point. Not sure if disabling the list is necessary anymore. I'll test it out and update with what I find.

Reply:
Seems to not make a difference. I'll remove it.

Reply:
https://github.com/dimagi/commcare-hq/pull/34987/commits/e0d69ef659fe359304ef97705a94640082912966

Reply:
Cool. I am just confused how this working.

we updated the default list path to get. So, the list one is still accessible? even if we had `list_allowed_methods = []` or now removed.

Reply:
> So, the list one is still accessible? even if we had list_allowed_methods = [] or now removed.

That's indeed what my testing confirmed

Reply:
Okay :+1: